src/background.js

```diff
@@ -18,12 +18,31 @@ chrome.alarms.onAlarm.addListener(function (alarm) {
           (injectionResults) => {
             for (const frameResult of injectionResults) {
               if (frameResult.result === "Devin is waiting") {
-                chrome.notifications.create({
-                  type: "basic",
-                  iconUrl: "icon.png",
-                  title: "Devin Status",
-                  message: "Devin is awaiting your response.",
-                });
+                // Retrieve the last notification time for the current tab from storage
+                chrome.storage.local.get(
+                  `lastNotificationTime_${tab.id}`,
+                  function (data) {
+                    const currentTime = new Date().getTime();
```

Comment:
I want you to use the time from time of each session last message. It would be "Mon 11:34 PM" from the following html example.

```
<!-- Message History -->
<div class="message-history--item pb-4 pt-4">
   <div class="message-item w-full"></div>
   <div class="message-item w-full"></div>
   <div class="message-item w-full">
      <div class="sender-text-icon"><img alt="Devin" loading="lazy" width="40" height="40" decoding="async" data-nimg="1" src="/devin_v4.png" style="color: transparent;"></div>
      <div class="message-item--content w-full">
         <div class="flex cursor-pointer items-center space-x-2"><span class="font-bold dark:text-white">Devin</span><span class="text-neutral-400" data-state="closed">Mon 11:34 PM</span></div>
         <div class="prose prose-sm dark:prose-invert prose-p:whitespace-pre-wrap prose-pre:whitespace-pre-wrap max-w-none [overflow-wrap:anywhere]">
            <p>Absolutely, I'll adjust the HTML to make sure the checkbox reflects the actual state from the storage. I'll remove the static 'checked' attribute from the HTML input element and rely on the JavaScript to set the correct state based on the user's previous choice. This way, the UI will accurately represent the stored state. I'll get this done and update you shortly.</p>
         </div>
      </div>
   </div>
</div>
```